crest/Assets/Crest/Crest/Scripts/Underwater/RegisterUnderwaterInput.cs

```diff
@@ -0,0 +1,43 @@
+// Crest Ocean System
+
+// This file is subject to the MIT License as seen in the root of this folder structure (LICENSE)
+
+namespace Crest
+{
+    using System.Collections.Generic;
+    using UnityEngine;
+
+    // TODO: Maybe rename.
+    public class RegisterUnderwaterInput : MonoBehaviour
```

Comment:
Yeah I think it may want to be distinct from the register input naming as it's quite different. Although that might just be  my bias from knowing the internals. My brain initially went to something like "ApplyUnderwaterEffectToAlpha"

Reply:
I renamed to _ApplyUnderwaterEffectToTransparent_ as that might be more recognisable.